xc7/environment.yml

```diff
@@ -2,8 +2,8 @@ name: xc7
 channels:
   - symbiflow
 dependencies:
-  - symbiflow::symbiflow-yosys=0.8_3925_g6bccd35a
-  - symbiflow::symbiflow-yosys-plugins=1.0.0.7_0032_g104f4fc
+  - symbiflow::symbiflow-yosys=0.8_6021_gd8b2d1a2
+  - symbiflow::symbiflow-yosys-plugins=1.0.0.7_0117_g160b309
```

Comment:
Shouldn't this match https://github.com/SymbiFlow/symbiflow-arch-defs/blob/8b9105d2009bbd9e59282b331d34187e65891110/environment.yml#L7 ?

Reply:
Fixed.